cmd/loom/gateway/general_cmds.go

```diff
@@ -657,6 +663,68 @@ func newSetWithdrawFeeCommand() *cobra.Command {
 	return cmd
 }
 
+func newSetWithdrawLimitCommand() *cobra.Command {
+	var totalLimit, accountLimit, decimals int64
+	cmd := &cobra.Command{
+		Use:     "set-withdrawal-limit <gateway>",
+		Short:   "Sets maximum amount the gateway should allow withdrawal",
+		Example: setWithdrawLimitCmdExample,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			loomKeyPath := gatewayCmdFlags.PrivKeyPath
+			hsmPath := gatewayCmdFlags.HSMConfigPath
+			algo := gatewayCmdFlags.Algo
+			signer, err := cli.GetSigner(loomKeyPath, hsmPath, algo)
+			if err != nil {
+				return err
+			}
+
+			var name string
+			if len(args) == 0 || strings.EqualFold(args[0], GatewayName) {
+				name = GatewayName
+			} else if strings.EqualFold(args[0], LoomGatewayName) {
+				name = LoomGatewayName
+			} else if strings.EqualFold(args[0], BinanceGatewayName) {
+				name = BinanceGatewayName
+			} else {
+				return errors.New("only Gateway, LoomCoin, or Binance gateway is allowed to set max withdrawal limit")
+			}
+
+			if totalLimit < 0 || accountLimit < 0 {
+				return errors.New("amount must be greater than zero")
+			}
+			if decimals < 0 {
+				return errors.New("decimals must be greater than zero")
+			}
+
+			// create amounts with decimals
+			maxTotalAmount := sciNot(totalLimit, decimals)
+			maxPerAccountAmount := sciNot(accountLimit, decimals)
+
+			rpcClient := getDAppChainClient()
+			gatewayAddr, err := rpcClient.Resolve(name)
+			if err != nil {
+				return errors.Wrap(err, "failed to resolve DAppChain Gateway address")
+			}
+			gateway := client.NewContract(rpcClient, gatewayAddr.Local)
+
+			req := &tgtypes.TransferGatewaySetMaxWithdrawalLimitRequest{
+				MaxTotalDailyWithdrawalAmount:      &types.BigUInt{Value: *maxTotalAmount},
+				MaxPerAccountDailyWithdrawalAmount: &types.BigUInt{Value: *maxPerAccountAmount},
+			}
```

Comment:
If I only specify one of the limits on the command line then the other will get reset to zero, that shouldn't happen, only the one I specify should change. Fetch the current limits, apply the requested changes, then submit them back to the cluster.